cmake/schemes/url_sha1_avahi_autotools.cmake.in

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include("@HUNTER_SELF@/cmake/Hunter")
+
+include(hunter_fatal_error)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+include(hunter_autotools_project)
+
+hunter_status_debug("Scheme: url_sha1_autotools")
+
+if("@MSVC@")
+  hunter_fatal_error("Autotools scheme not supported with Visual Studio")
+endif()
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_BUILD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+
+hunter_add_package(Expat)
+hunter_add_package(libdaemon)
+hunter_add_package(intltool)
+
+hunter_autotools_project("@HUNTER_EP_NAME@"
+    HUNTER_SELF
+      "@HUNTER_SELF@"
+    URL
+      @HUNTER_PACKAGE_URL@
+    URL_HASH
+      SHA1=@HUNTER_PACKAGE_SHA1@
+    DOWNLOAD_DIR
+      "@HUNTER_PACKAGE_DOWNLOAD_DIR@"
+    SOURCE_DIR
+      "@HUNTER_PACKAGE_SOURCE_DIR@"
+    BUILD_DIR
+      "@HUNTER_PACKAGE_BUILD_DIR@"
+    INSTALL_DIR
+      "@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    PARALLEL_JOBS
+      "@HUNTER_JOBS_OPTION@"
+    EXTRA_FLAGS
+     --disable-qt3
+     --disable-qt4
+     --disable-gtk
+     --disable-gtk3
+     --disable-gdbm
+     --disable-python
+     --disable-mono
+     --disable-monodoc
+     --with-xml=expat
+     --enable-compat-libdns_sd
```

Comment:
> it would be smart to do it in toolchain.cmake

I think no, toolchain is a "global" object to all of the packages. It means that same toolchain can be used for two different versions of `intltool`. Since versions is different they have a different locations: `<root-id-1>/Install/bin/intltool` (v1), `<root-id-2>/Install/bin/intltool` (v2).

> So in fact I've added the forwarding of the PATH variable within the hunter_autotools_project.cmake which allows the built intltool to be found

> What is however problematic with the forwarding of PATH as I did, is by cross-compiling one would like the intltool-native used to configure an avahi for another target, and this isn't handled. 

The best choice is to add a location explicitly (like `--intltool-path=/path/to/intltool`) if there is no such option in `configure` then we need an extra argument to the `hunter_autotools_project` (like `ENV_PATH`).

> Actually I'm unsure if you have the concept in hunter of a sysroot-native and sysroot-target, wouldn't it make sense ?

There is no any "special" concept in Hunter, it just inherit CMake concept of cross-compiling. I.e. you can use `CMAKE_HOST_SYSTEM_NAME` to detect Linux host platform or `ANDROID` and `IOS` to check you're builing Android or iOS targets.

> For the search of libraries via pkgconfig with autoreconf, I also added to the configure call the PKG_CONFIG_PATH so that libraries get searched in HUNTER_PACKAGE_INSTALL_PREFIX

Probably something similar can be applied to the PATH, what do you think?